core/db/migrate/20210114220232_remove_store_id_from_spree_payment_methods.rb

```diff
@@ -1,5 +1,20 @@
 class RemoveStoreIdFromSpreePaymentMethods < ActiveRecord::Migration[6.0]
+
+  def up
+    Spree::PaymentMethod.all.each do |payment_method|
+       if payment_method.store_id.nil?
+         Spree::Store.all.each do |store|
+           payment_method.store_ids = store.id
+           payment_method.save
+         end
+       else
+         payment_method.store_ids = [:store_id]
+         payment_method.save
+       end
+     end
+  end
+
   def change
-    remove_column :spree_payment_methods, :store_id
+    remove_column :spree_payment_methods, :store_id, :integer
```

Comment:
@MatthewKennedy I think we don't need that `, :integer` option at the end

Reply:
I'll take that out when finished, but rolling back and re-migrating to test the data migration code as I work, and it won't roll back unless it specifies type on the column.

I pushed that last commit so I can move from working on a laptop to desktop, nothing in that last push is final.